liteapi/client.go

```diff
@@ -510,21 +510,53 @@ func (c *Client) GetTransactions(
 		return nil, err
 	}
 	if len(r.Transactions) == 0 {
-		return []tlb.Transaction{}, nil
+		return []tongo.Transaction{}, nil
 	}
 	cells, err := boc.DeserializeBoc(r.Transactions)
 	if err != nil {
 		return nil, err
 	}
-	var res []tlb.Transaction
-	for _, cell := range cells {
+	var res []tongo.Transaction
+	for i, cell := range cells {
 		var t tlb.Transaction
 		cell.ResetCounters()
 		err := tlb.Unmarshal(cell, &t)
 		if err != nil {
 			return nil, err
 		}
-		res = append(res, t)
+		res = append(res, tongo.Transaction{
+			Transaction: t,
+			BlockID:     r.Ids[i].ToBlockIdExt(),
+		})
+	}
+	return res, nil
+}
+
+func (c *Client) GetLastTransactions(ctx context.Context, a tongo.AccountID, limit int) ([]tongo.Transaction, error) {
+	state, err := c.GetAccountState(ctx, a)
+	if err != nil {
+		return nil, err
+	}
+	lastLt, lastHash := state.LastTransLt, state.LastTransHash
+	var res []tongo.Transaction
+	for {
+		txs, err := c.GetTransactions(ctx, 10, a, lastLt, tongo.Bits256(lastHash))
+		if err != nil {
+			if e, ok := err.(liteclient.LiteServerErrorC); ok && int32(e.Code) == -400 {
```

Comment:
I think, it might be helpful to add a comment about what this code and error mean